yarn.lock

```diff
@@ -7610,7 +7610,7 @@ ses@^0.12.6:
     "@agoric/make-hardener" "^0.1.2"
     "@agoric/transform-module" "^0.4.1"
 
-ses@^0.14.1:
+ses@^0.14.1, ses@^0.14.2:
```

Comment:
What is entraining the prior version? Can you `yarn why ses`?

Reply:
The prior version comes from other published `@agoric/...` dependencies in the `yarn.lock`.  Each dapp currently is expected to be able to function even if you don't use `agoric install`, but use `yarn install` to get all its dependencies.  So, the `yarn.lock` typically drifts a little bit behind what the current `agoric-sdk` has, until the SDK has been published and the dapp's `yarn.lock` can be updated.